atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/stream/StreamTestSchema.java

```diff
@@ -51,6 +51,7 @@ private static Schema generateSchema() {
         // test defaults
         schema.addStreamStoreDefinition(
                 new StreamStoreDefinitionBuilder("stream_test", "stream_test", ValueType.VAR_LONG)
+                        .hashRowComponents()
```

Comment:
A bit worried that you loose testing on the default case here. This should ideally be a new `addStreamStoreDefinition` and we can parameterize the tests to run on both the tables.